src/Configuration/CliOptions.cs

```diff
@@ -71,7 +71,23 @@ public static (PlcSimulation PlcSimulationInstance, List<string> ExtraArgs) Init
                         throw new OptionException("The max OPC string length must be larger than 0.", "opcmaxstringlen");
                     }
                 }
-            },
+            },
+
+            // OTLP Exporter Configuration
+            { "otlpen|otlpenabled", $"enables the otlp exporter.\nDefault: {config.OtlpEnabled}", (s) => config.OtlpEnabled = s != null },
```

Comment:
>otlpen [](http://example.com/codeflow?start=15&length=6)

Do we need this to be enabled/disabled or could we use setting the endpoint as a way to enable it?

Reply:
(depends on the requirements)